app/frontend/angular.json

```diff
@@ -25,7 +25,8 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.css"
+              "src/styles.css",
+              "node_modules/material-design-icons/iconfont/material-icons.css"
```

Comment:
"node_modules/material-design-icons/iconfont/material-icons.css" file creates an error. please remove it